infra/grafana/provisioning/dashboards/my-dashboard.json

```diff
@@ -0,0 +1,189 @@
+{
```

Comment:
Btw this was previously generated somehow?

Reply:
Probably crafted in Grafana and copied as a json.

Reply:
Yup exactly this way as @PiotrWodecki said.